<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="com.framework.utils.RetryListener"/>
  </listeners>
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="com.trulances.testcases.TC_14007_VerifySubheadofHeader2">
        <methods>
          <include name="endResult"/>
          <include name="stopTestCase"/>
          <include name="preCondition"/>
          <include name="startReport"/>
          <include name="postCondition"/>
          <include name="runVerifySubheadofHeader2" invocation-numbers="0 0"/>
          <include name="setValues"/>
          <include name="startTestCase"/>
        </methods>
      </class> <!-- com.trulances.testcases.TC_14007_VerifySubheadofHeader2 -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
